buf.gen.yaml

```diff
@@ -8,9 +8,9 @@ managed:
       - buf.build/googleapis/googleapis
 
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/go
+  - plugin: buf.build/protocolbuffers/go:v1.30.0
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.

Reply:
ps. you can omit the version and it'll use the latest by default. e.g.,

```
plugins:
  - plugin: buf.build/protocolbuffers/go
...
```